controls/srg_gpos/SRG-OS-000064-GPOS-00033.yml

```diff
@@ -22,16 +22,7 @@ controls:
             - audit_rules_unsuccessful_file_modification_ftruncate
             - audit_rules_unsuccessful_file_modification_open
             - audit_rules_unsuccessful_file_modification_open_by_handle_at
-            - audit_rules_unsuccessful_file_modification_open_by_handle_at_o_creat
-            - audit_rules_unsuccessful_file_modification_open_by_handle_at_o_trunc_write
-            - audit_rules_unsuccessful_file_modification_open_by_handle_at_rule_order
-            - audit_rules_unsuccessful_file_modification_open_o_creat
-            - audit_rules_unsuccessful_file_modification_open_o_trunc_write
-            - audit_rules_unsuccessful_file_modification_open_rule_order
             - audit_rules_unsuccessful_file_modification_openat
-            - audit_rules_unsuccessful_file_modification_openat_o_creat
-            - audit_rules_unsuccessful_file_modification_openat_o_trunc_write
-            - audit_rules_unsuccessful_file_modification_openat_rule_order
             - audit_rules_unsuccessful_file_modification_rename
```

Comment:
In the newest STIGs these are combined, how do we want to handle this?

Reply:
We will push them to keep rules separate, if they ultimately reject we can reevaluate.